libsel4/sel4_arch_include/aarch64/interfaces/sel4arch.xml

```diff
@@ -207,4 +217,18 @@
             </error>
         </method>
     </interface>
+    <interface name="seL4_ARM_SMC" manual_name="SMC" cap_description="Capability to allow threads to interact with SMC calls.">
+        <method id="ARMSMCCall" name="Call" manual_name="SMC Call" manual_label="smc_call">
+            <brief>
+                Tell the microkernel to make the real SMC call.
+            </brief>
+            <description>
+                <docref>See <autoref label="sec:read_write_registers"/></docref>
```

Comment:
Is this reference still up-to-date, considering that `seL4_ARM_SMCContext` has been added?

Reply:
Updated the description accordingly.